README.md

```diff
@@ -1,5 +1,28 @@
 # Deployment Guide
 
+The retail industry is modernizing their applications, many of which are latency sensitive, need high availability and would need distributed edge computing capabilities in order to support them. Azure Edge Zones combined with Couchbase’ highly scalable, distributed and embedded database can provide the perfect platform for such applications.
+
+In this deployment guide we will detail how to deploy the architecture in Microsoft Azure to support this retail use case. We will be deploying the following resources:
```

Comment:
Make similar changes here? How about something like: In this deployment guide we will detail how to deploy Couchbase in Microsoft Azure. The Web application and database will use retail as an example.